tests/annotations/tests.py

```diff
@@ -148,6 +148,19 @@ def test_mixed_type_annotation_numbers(self):
         combined = int(test.pages + test.rating)
         self.assertEqual(b.combined, combined)
 
+    def test_empty_expression_annotation(self):
+        books = Book.objects.annotate(
+            selected=ExpressionWrapper(Q(pk__in=[]), output_field=BooleanField())
```

Comment:
I would prefer it to be removed since it's not something we've committed to supporting. The problem is that there are many people already using Q in this way rather than the longer but more correct Case/When. I'm fairly sure wrapped Qs will break in subtle ways but I haven't dug into it enough to come up with a breaking scenario.